articles/2020-04-02-scaling-stories-comment-les-startups-se-sont-plantees.md

```diff
@@ -0,0 +1,146 @@
+---
+date: 2020-04-02
+title: Scaling stories - comment les startups se sont plantées
+author: cyppan
+slug: scaling-stories-comment-les-startups-se-sont-plantees
+---
+
+> - "Ca ne fonctionne pas, mes enfants n'arrivent pas à se connecter à votre service !"
+> - "Nous sommes désolé notre équipe technique est mobilisée à 100% sur ce problème c'est une situation exceptionnelle... Nous vous tenons au courant."
+
+
+C'est le genre de tweet ou de message Linkedin qu'on a pu voir se multiplier avec le confinement récent dû au Covid-19. En effet les services web, et en particulier les outils de communication ou les programmes du secteur de l'edtech (education technology), ont dû faire face à un usage particulièrement intense et soutenu ces derniers temps. Il n'y en a que peu à ma connaissance qui ont été capables de maintenir une qualité de service acceptable (Slack et Zoom ont particulièrement brillé sur ce sujet), et la plupart a dû mettre en place des solutions minutes comme des queues de connexion, 
+d'autres encore ont été forcés de désactiver leur service pour une "maintenance" à durée indéterminée...
+
+
+Même en étant compréhensifs, face à un service planté, on a vite fait de changer. Après tout, avec le choix plus que fourni de services dont on dispose, pourquoi se priver ? Le problème c'est qu'après la crise, il est fort probable que nous ne reviendrons pas au service qu'on avait pourtant trouvé si cool au premier abord... Aïe, pour une startup c'est beaucoup de clients perdus, peut-être assez pour ne jamais retrouver de la traction. Une mort pénible donc pour un business, victime lui aussi du virus.
+
+
+Mais 🧐
+
+* *Ces entreprises n'auraient-elles pas pu mieux anticiper ?*
+* *N'auraient-elle pas pu avoir une meilleure tech à la base, crise ou pas crise ?*
+* *Et une fois devant le problème, n'auraient-elles pas pu mieux s'adapter et plus rapidement ?*
+
+
+## le modèle de "scaling maturity"
+
+
+![modèle de scaling maturity](https://cyppan.me/scaling-maturity-fr.png)
```

Comment:
Can you please bring that image to the repository so that it doesn't break if you change it on your server?